libexec/rtld-elf/rtld_c18n.c

```diff
@@ -530,17 +521,23 @@ _rtld_setjmp_impl(uintptr_t ret, void **buf, struct trusted_frame *csp)
 	 * buffer.
 	 */
 
-	*buf = cheri_seal(cheri_setaddress(csp, csp->next), sealer_jmpbuf);
+	/*
+	 * Defensive reduction of privileges.
+	 */
+	csp = cheri_setaddress(csp, csp->next);
+	csp = cheri_setboundsexact(csp, 0);
+	csp = cheri_andperm(csp, 0);
```

Comment:
If it's sealed what's the point of bounds and permissions manipulation? And why is it part of this change?